ui/src/components/form/checkbox-field.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import React from 'react';
+import {
+  FieldPathByValue,
+  FieldPathValue,
+  FieldValues,
+  UseFormReturn,
+} from 'react-hook-form';
+
+import { Checkbox } from '@/components/ui/checkbox';
+import {
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { Label } from '@/components/ui/label';
+import { Separator } from '@/components/ui/separator';
+
+type CheckboxFieldProps<
+  TFieldValues extends FieldValues,
+  TName extends FieldPathByValue<TFieldValues, Array<string>>,
+> = {
+  form: UseFormReturn<TFieldValues, TName>;
+  name: TName;
+  label?: string;
+  description?: React.ReactNode;
+  items: readonly { id: string; label: string }[];
+};
+
+export const CheckboxField = <
```

Comment:
This should have a more descriptive name, as `CheckboxField` doesn't convey it being a multiselect.